### DISCUSSION TITLE
I don't want this thing: expansion, How to hide?

### DISCUSSION THREAD

**Discussion by alkene5** [marlonrichert/zsh-autocomplete#795] [Q&A] [answered]
<img width="525" alt="image" src="https://github.com/user-attachments/assets/cd78fc4d-e6da-4585-83a4-c1d0ea76f6a9">

Sorting is also possible, Don't put it on top anyway 🙁

**Comment by alitajelsir** [comment, marked as answer]
- By default `zsh-autocomplete` sets `completer functions` to:
  https://github.com/marlonrichert/zsh-autocomplete/blob/762afacbf227ecd173e899d10a28a478b4c84a3f/Functions/Init/.autocomplete__config#L13-L14

- Set whatever `completer functions` you want in `.zshrc` after you  source `zsh-autocomplete`

- In this case remove the `_expand` function:
  
  ```
  zstyle ':completion:*' completer _complete _complete:-fuzzy _correct _approximate _ignored
  ```
